library/core/src/panic/panic_info.rs

```diff
@@ -62,6 +65,12 @@ impl<'a> PanicInfo<'a> {
         self.payload = info;
     }
 
+    #[unstable(feature = "error_in_panic", issue = "none")]
+    /// The [`Error`] that caused this panic, if known.
+    pub fn source(&self) -> Option<&(dyn Error + 'static)> {
+        self.source
```

Comment:
```suggestion
    pub fn error(&self) -> Option<&(dyn Error + 'static)> {
        self.error
```
(etc.)

See comment above.